roles/pulp3/tasks/install.yml

```diff
@@ -52,19 +52,36 @@
 
 - block:
 
-    - name: Install pulpcore
+    - name: Install pulpcore packages from source
       pip:
-        name: pulpcore
-        state: present
+        name: '{{ pulp_source_dir }}/{{ item }}'
+        editable: yes
         virtualenv: '{{ pulp_install_dir }}'
         virtualenv_command: '{{ pulp_python_interpreter }} -m venv'
+      with_items:
+        - common
```

Comment:
We talked about removing this, but I never commented on the PR.

Reply:
removed in next push